lib/pf/util.pm

```diff
@@ -1011,6 +1012,25 @@ sub cert_has_expired {
     return time > $expiration;
 }
 
+=item strip_username
+
+Will strip a username matching pattern user@realm or \\realm\user
+
+Returns ($user,$realm) if found or ($user) if not matching any realm pattern
+
+=cut
+
+sub strip_username {
```

Comment:
I think you will probably have to add $username =~ /(.*)\%(.*)/ to be close to radius default config